lib/capistrano/tasks/bundler.cap

```diff
@@ -63,8 +63,10 @@ namespace :load do
 
     set :bundle_roles, :all
     set :bundle_servers, -> { release_roles(fetch(:bundle_roles)) }
-    set :bundle_path, -> { shared_path.join('bundle') }
+    set :bundle_path, nil
     set :bundle_without, %w{development test}.join(' ')
     set :bundle_flags, '--deployment --quiet'
+
+    set :linked_dirs, fetch(:linked_dirs, []).push('vendor/bundle')
```

Comment:
Valid point, but we can't detect that, can we?
During the default-task we don't know yet if the user changed `bundle_path`.
And during the bundle-install-task it is too late to add a `linked_dir`...